Formula/lib/libchaos.rb

```diff
@@ -21,16 +21,22 @@ class Libchaos < Formula
 
   depends_on "cmake" => :build
 
+  # Support for Xcode 15+ (LLVM 16+)
+  patch :DATA
```

Comment:
should we upstream this patch?

Reply:
If @Bo98 wants to since it is copy of above comment. I don't expect any more upstream activity given 8 years since last activity.

Reply:
still worth poking though :)